setup.py

```diff
@@ -23,6 +18,15 @@ def read_version():
                 return line.split('=')[1].strip().strip("'")
 
 
+# get the dependencies and installs
+with open("requirements.txt", "r", encoding="utf-8") as f:
+    requires = [x.strip() for x in f if x.strip()]
+
+
+# get test dependencies and installs
+with open("test_requirements.txt", "r", encoding="utf-8") as f:
```

Comment:
How does it affect the package? Package should contain minimal dependencies and should not include tests

Reply:
https://godatadriven.com/blog/a-practical-guide-to-using-setup-py/